Dockerfile_multi_folder

```diff
@@ -0,0 +1,14 @@
+ARG BASE_IMAGE=nansencenter/geospaas:2.0.0-slim
+FROM ${BASE_IMAGE}
+RUN pip install --no-cache-dir django-celery-results==1.2
+WORKDIR /tmp/setup
+COPY setup.py README.md ./
+COPY geospaas_processing ./geospaas_processing
+RUN python setup.py bdist_wheel && \
+    pip install -v dist/geospaas_processing-*.whl && \
```

Comment:
That can can probably be replaced by `python setup.py install` (needs testing).
I don't remember why I built the wheel instead of installing directly in the other Dockerfile.

Reply:
Done in the next commit.